File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/D. Fixed Password/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/D

using System;

namespace Codeforces.Assuit.Contest1
{
    class ProblemD
    {
        static void Main(string[] args)
        {
            ushort password;
            while(true){
                password = ushort.Parse(Console.ReadLine());
                if(password == 1999) break;
                Console.WriteLine("Wrong");
            }
            Console.WriteLine("Correct");
        }
    }
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/D. Fixed Password/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/D
#include<iostream>
#include<cstdio>

using namespace std;
 
int main(){
    unsigned short password;
    while(true){
        cin >> password;
        if (password == 1999) break;
        cout << "Wrong\n";
    }
    cout << "Correct\n";
}